src/components/timeline/index.js

```diff
@@ -203,6 +203,14 @@ class TimelineSettingsActiveTabView extends React.PureComponent<{|
 }
 
 class Timeline extends React.PureComponent<Props> {
+  state = {
+    intialSelected: null,
+  };
+
+  setInitialSelected = (el: HTMLElement) => {
```

Comment:
I think this scrolling mechanism could use a bit of documentation. Generally this project uses lots of documentation to explain why code exists and what it's attempting to do. This method seems like a good place to provide this documentation:

```
/**
 * This method collects the initially selected track's HTMLElement. This allows the timeline
 * to scroll the initially selected track into view once the page is loaded.
 */
```